paper-bottom-sheet.html

```diff
@@ -105,6 +105,16 @@
 				reflectToAttribute: true
 			},
 
+			/**
+			* Set to true to disable canceling the overlay with the ESC key.
+			*/
+			noCancelOnEscKey: {type: Boolean, value: false},
```

Comment:
Thanks for your contribution, @srijken Could you please spread out the property definition over multiple line to keep the code consistent:
```
noCancelOnEscKey: {
  type: Boolean,
  value: false
},
```

Afterwards, pls squash this commit together with https://github.com/Collaborne/paper-bottom-sheet/pull/10/commits/9bfce223d334f132b87b938e6b4f368e4bfa9509 (good to keep the `Add .gitignore` in a separated commit).
Thanks!